src/app.module.js

```diff
@@ -1,7 +1,11 @@
 import angular from 'angular';
 import '@uirouter/angularjs';
+import 'angular-translate';
+import 'angular-sanitize';
+import 'angular-translate-loader-static-files';
 
-angular.module('core', ['templates', 'ui.router', 'searchPage', 'shared', 'home']);
+angular.module('core', ['templates', 'ui.router', 'searchPage', 'shared', 'home', 'pascalprecht.translate', 'translate', 'ngSanitize']);
```

Comment:
Please make this array vertical for better readability

Reply:
Updated.